### ISSUE TITLE
Problem in reading data

### ISSUE THREAD

**Issue by AlenaLangova** [mzmine/mzmine#1571]
### Basic information

* My operating system and version: Wind
* My MZmine version: MZmine 3.9.0

Hi,
I have just started working with MZmin according to your YouTube videos, but there is a problem that I cannot solve. I am sending a screenshot of the error in the attachment. it's about the fact that I can upload data, I can see the chromatograph and the masses, but for some time the window pops up at me. Could you please advise me?

![Snímek obrazovky 2024-01-03 133321](https://github.com/mzmine/mzmine3/assets/155546684/224b017e-2563-4586-a07f-8780d30da77d)

**Comment by ansgarkorf** [comment]
Hi Alena,
Looks like the mzML data you are using dies not have its scans sorted by retention time. 
What instrument did you use?
How did you convert the data?
Can you provide this mzML file for testing? 
Ansgar

**Comment by AlenaLangova** [comment]
Hi,
 I normally used  HILIC/QTOF as a instrument and as for the convert a used
MSConvertGUI. I am from the Michal Holčapek group and few people of this
group worked with old version, but they still do not know.

Best,
Alča

st 3. 1. 2024 v 13:52 odesílatel Ansgar Korf ***@***.***>
napsal:

**Comment by ansgarkorf** [comment]
Hi,
I need a little bit more info on the QTOF, vendor and instrument specification would be good. If you could share the mzML file I can have a look myself.
kind regards,
Ansgar

**Comment by AlenaLangova** [comment]
Where can I share you the file?

A.

st 3. 1. 2024 v 14:07 odesílatel Ansgar Korf ***@***.***>
napsal:

**Comment by ansgarkorf** [comment]
You can use any data sharing tool you like, e.g. google drive.

**Comment by AlenaLangova** [comment]
Do you see it on google drive?

A.

st 3. 1. 2024 v 14:26 odesílatel Ansgar Korf ***@***.***>
napsal:

**Comment by ansgarkorf** [comment]
Sorry, I dont understand. Did you share a google drive link somewhere? You can share the link here or send it to my email:
ansgar.korf@mzmine.com

**Comment by AlenaLangova** [comment]
I send it.
A

st 3. 1. 2024 v 14:49 odesílatel Ansgar Korf ***@***.***>
napsal:

**Comment by ansgarkorf** [comment]
yes, I got it and will have a look at the file

**Comment by ansgarkorf** [comment]
I was able to reproduce the issue and I found some strange scan numbering in the raw file. See screenshot:
![Screenshot 2024-01-03 at 16 20 46](https://github.com/mzmine/mzmine3/assets/27132602/fd42187c-ebc7-4b91-acfe-dbb0846a81f7)

Since you didnt clarify the MS vendor yet, I make a wild guess and assume you are using a Waters QTOF. If this is the case please check out our documentation for raw data conversion:
https://mzmine.github.io/mzmine_documentation/data_conversion.html

Waters has recently released a tool to convert into mzML:
https://microapps.on-demand.waters.com/home/showmarkdown/data-as-a-product

I hope this helps.

**Comment by ansgarkorf** [comment]
Dear Alena,
did you have more success with the suggested converter?
kind regards,
Ansgar

**Comment by AlenaLangova** [comment]
Hi,

I apologize for my delayed response. Honestly, I didn't try the converter
as we found an alternative solution for our identification needs. However,
it's likely that we will require MZmine in the future. I want to express my
gratitude for your time, and perhaps, I may reach out to you again in the
future. Following a discussion with Professor Holčapek, I have enrolled in
your April workshop.

Best regards,
Alena

pá 5. 1. 2024 v 6:56 odesílatel Ansgar Korf ***@***.***>
napsal:
